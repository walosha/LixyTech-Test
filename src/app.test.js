import React from "react";
import ReactDom from "react-dom";
import { render } from "@testing-library/react";
import TestRenderer from "react-test-renderer";

import App from "./app";

it("Render Button Successfully ", () => {
  const div = document.createElement("div");
  ReactDom.render(<App />, div);
});

it("Render Button Successfully ", () => {
  const tree = TestRenderer.create(<App />).toJSON();
  expect(tree).toMatchSnapshot();
});

describe("HOME TEST", () => {
  test("renders Table with #", () => {
    const { getByText } = render(<App />);
    const linkElement = getByText(/#/i);
    expect(linkElement).toBeInTheDocument();
  });
});
