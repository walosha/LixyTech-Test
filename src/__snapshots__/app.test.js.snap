// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render Button Successfully  1`] = `
<div
  style={
    Object {
      "display": "flex",
      "flexDirection": "column",
      "padding": "20px",
    }
  }
>
  <div
    className="inputContainer"
  >
    <input
      onChange={[Function]}
      placeholder="Enter a number"
      type="text"
    />
  </div>
  <table
    className="table table-striped"
  >
    <thead>
      <tr>
        <th>
          #
        </th>
      </tr>
    </thead>
    <tbody />
  </table>
</div>
`;
